test-cases/tests/walkthroughs/g15-verify-soluton-explorer-customer-settings-backup.md

```diff
@@ -0,0 +1,20 @@
+---
+estimate: 15m
+tags:
+  - 2.5.0
+---
+
+# G15 - Verify soluton explorer customer settings backup
+
+Acceptance Criteria:
+
+When logged into solution explorer as a customer-admin
+
+1. Select ? in the top right hand corner to check the Solution Explorer version. It should be 2.26.1.
```

Comment:
should it not be more generic, e.g. 

Select ? in the top right hand corner to check the Solution Explorer version and confirm that it matches the version obtained via CLI: `oc get rhmi rhmi -n redhat-rhmi-operator -o json | jq -r '.status.stages."solution-explorer".products."solution-explorer".version'`

Reply:
Agreed